fedorov/fedorov.py

```diff
@@ -202,6 +204,103 @@ def translate_to_vector_2D(a=1, b=1, theta=np.pi / 2):
     return lattice_vectors
 
 
+def populate_system(
+        lattice: np.ndarray,
+        basis_vectors: np.ndarray,
+        num_replicas: typing.Union[int, np.ndarray],
+        center: bool = True,
+        **kwargs: np.ndarray):
```

Comment:
I don't think `**kwargs` is an array. Probably drop this type annotation, I'm not even sure if it's sensible to call it a dict.